minfraud/request.py

```diff
@@ -13,7 +13,7 @@
 from voluptuous import MultipleInvalid
 
 from .errors import InvalidRequestError
-from .validation import validate_report, validate_transaction
+from .validation import _validate_report, _validate_transaction
```

Comment:
Sorry, I didn't mean that we should add an underscore to these. I just meant the intermediate functions that you added that are only used in that file.